plugin/pkg/admission/storage/persistentvolume/label/admission_test.go

```diff
@@ -232,6 +233,136 @@ func Test_PVLAdmission(t *testing.T) {
 			},
 			err: nil,
 		},
+		{
+			name:    "existing labels from dynamic provisioning are not changed",
+			handler: newPersistentVolumeLabel(),
+			pvlabeler: mockVolumeLabels(map[string]string{
+				v1.LabelZoneFailureDomain: "domain1",
+				v1.LabelZoneRegion:        "region1",
+			}),
+			preAdmissionPV: &api.PersistentVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "awsebs", Namespace: "myns",
+					Labels: map[string]string{
+						v1.LabelZoneFailureDomain: "existingDomain",
+						v1.LabelZoneRegion:        "existingRegion",
+					},
+					Annotations: map[string]string{
+						persistentvolume.AnnDynamicallyProvisioned: "true",
+					},
+				},
+				Spec: api.PersistentVolumeSpec{
+					PersistentVolumeSource: api.PersistentVolumeSource{
+						AWSElasticBlockStore: &api.AWSElasticBlockStoreVolumeSource{
+							VolumeID: "123",
+						},
+					},
+				},
+			},
+			postAdmissionPV: &api.PersistentVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "awsebs",
+					Namespace: "myns",
+					Labels: map[string]string{
+						v1.LabelZoneFailureDomain: "existingDomain",
+						v1.LabelZoneRegion:        "existingRegion",
+					},
+					Annotations: map[string]string{
+						persistentvolume.AnnDynamicallyProvisioned: "true",
```

Comment:
In the admission controller we only check existence, is that true for other consumers of this annotation? Asking cause the value is "true" and was wondering if that was arbitrary or intentional.

Reply:
* It is used by PV controller
* And any nozero value is valid here (technically, it's a name of provisioner).  Maybe I should have chosen a better value

Reply:
changed to `kubernetes.io/aws-ebs`, thanks @msau42 for holding the PR